src/styles/index.scss

```diff
@@ -1,3 +1,122 @@
-body {
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap';
+
+.card {
+  width: 200px;
   margin: 0;
+  font-family: Roboto,sans-serif;
+  font-weight: 400;
+  display: flex;
+  flex-direction: column;
+  justify-content:center;
+  box-sizing: border-box;
+  border: 1px solid #F3F3F3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    margin-left: 3px;
+    max-width: 160px;
+    height: 134px;
+    box-sizing: border-box;
+  }
+
+  &__discription {
+   color: #060B35;
+   padding-top: 40px;
+   font-size: 12px;
+   font-weight: 500;
+   line-height: 18px;
+   letter-spacing: 0;
+   height: 36px;
+  }
+
+  &__prodact {
+    height: 14px;
+    color: #616070;
+    font-size: 10px;
+    padding-top: 4px;
+    margin-inline-start: 0;
+  }
+
+  &__review {
+    display: flex;
+    align-items: center;
+    padding-top: 16px;
+
+    &-stars {
+      display: flex;
+
+      &-star {
+        width: 16px;
+        height: 16px;
+        flex-shrink: 0;
+        margin-right: 4px;
+        background-repeat: no-repeat;
+        background-position: center;
+        background-image: url(../images/star.svg);
+        &:nth-child(-n + 4) {
+          background-image: url(../images/star-active.svg);
+        }
+      }
```

Comment:
Move stars styles to the separate file

Reply:
<img width="613" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/375bcc5d-4648-4225-bff1-1b1a601f45f9">

Reply:
it is in separate file...